cups/ipp.c

```diff
@@ -34,13 +34,13 @@ static char		*ipp_lang_code(const char *locale, char *buffer, size_t bufsize) _C
 static size_t		ipp_length(ipp_t *ipp, int collection);
 static ssize_t		ipp_read_http(http_t *http, ipp_uchar_t *buffer,
 			              size_t length);
-static ssize_t		ipp_read_file(int *fd, ipp_uchar_t *buffer,
+static ssize_t		ipp_read_file(const int *fd, ipp_uchar_t *buffer,
```

Comment:
The ippRead/WriteIO callbacks don't use const because some IO mechanisms will necessarily change the contents of the pointed to data. Strictly speaking, the const-ness needs to match for C99, so we shouldn't change these functions.